src/uzfs_prop.c

```diff
@@ -41,3 +41,38 @@ uzfs_zinfo_update_rdonly(const char *name, const char *val)
 	nvlist_free(props);
 	return (error);
 }
+
+int
+uzfs_zpool_rdonly_cb(const char *ds_name, void *arg)
```

Comment:
why do we need this? do we need to update the cache for every child if readonly is set on the pool?

Reply:
yes. We require this to disable management connection for each zvol.

Reply:
can we set reconnect delay to infinity in case of readony is set on the pool, this way we can avoid traversing of each children?